src/utils/health-check.ts

```diff
@@ -90,30 +108,36 @@ const fetchProfileOrExit = async (logBalances: Function): Promise<OnchainProfile
   return profile;
 };
 
-// fix this logic, add curse fee.
 const warnIfFreeBondIsLessThanMinDiggingFee = (
   freeBondBal: BigNumber,
-  minDiggingFee: BigNumber
+  curseFee: BigNumber,
+  diggingFeePerSecond: BigNumber
 ): void => {
-  if (freeBondBal.lt(minDiggingFee)) {
+  const diggingFeePerMonth = diggingFeePerSecond.mul(ONE_MONTH_IN_SECONDS).add(curseFee);
+
+  if (freeBondBal.lt(diggingFeePerMonth)) {
     archLogger.warn(
-      `\n   Your free bond is less than your minimum digging fee. You will not be able to accept new jobs!`,
+      `\n   Your free bond is less than you can lock for a new Sarcophagus lasting a month. You may not be able to accept new jobs!`,
       true
     );
-    archLogger.error(`   Run: \`cli update -f <amount>\` to deposit some SARCO\n`, true);
+    archLogger.warn(`   Run: \`cli update -f <amount>\` to deposit some SARCO\n`, true);
   }
 };
 
-export const warnIfEthBalanceIsLow = async (ethBalanceArg?: BigNumber): Promise<void> => {
-  const ethBalance = ethBalanceArg ?? (await getEthBalance());
+export const warnIfEthBalanceIsLow = async (
+  doNotify?: boolean
+): Promise<{ ethBalance: BigNumber }> => {
+  const ethBalance = await getEthBalance();
   if (ethBalance.lte(ethers.utils.parseEther("0.05"))) {
     archLogger.error(
       `\nYou have very little ETH in your account: ${ethers.utils.formatEther(
         ethBalance
       )} ETH.\nYou may not have enough gas for any transactions!\n`,
-      true
+      { logTimestamp: true, sendNotification: doNotify }
```

Comment:
for terseness (so you dont need key/value), could make the argument here `sendNotification` instead of `doNotify`